CefSharp.Test/Framework/AsyncExtensionFacts.cs

```diff
@@ -12,6 +12,8 @@ namespace CefSharp.Test.Framework
     /// <summary>
     /// Async Extensions - This test doesn't need to be part of the 
     /// </summary>
+    //NOTE: All Test classes must be part of this collection as it manages the Cef Initialize/Shutdown lifecycle
+    [Collection(CefSharpFixtureCollection.Key)]
```

Comment:
Are you adding this for completeness? It shouldn't be required and actually hinder the performance of running these tests in isolation.

The comment should probably be updated to reflect that they need to be part of the collection when they access `Cef`, when they're purely managed tests no special requirements.

Reply:
This is required here because of the binding redirects which are now done using `AssemblyResolve` because the xunit runner which is used on appveyor does not support `app.config` binding redirects.
But I can also split the logic into a separate fixture.

This is the reason why the `CefSharp.Test.Framework.AsyncExtensionFacts.DeleteCookiesReturnsTaskWhenReturnsFalse` test failed https://ci.appveyor.com/project/cefsharp/cefsharp/builds/31114188/tests

Reply:
Are you using this to hook the assembly resolve?

Reply:
Thanks,  that would be preferred.